map/traffic_manager.hpp

```diff
@@ -28,6 +28,16 @@ class DrapeEngine;
 class TrafficManager final
 {
 public:
+  enum TrafficState
+  {
+    Disabled,
+    Enabled,
+    WaitingData,
+    Outdated,
+    NoData,
+    NetworkError
```

Comment:
Do we need this state?
Isn't it enough to set NoData and log the network error?